de_lon8/Kareem/Mini_Project_1/Week 1 simple.py

```diff
@@ -173,43 +176,80 @@
                 print (courier)
 
 
-    # if choice == 2:
-    #     while True:
-    #         print ('')
-    #         print ('')
-    #         print ('ORDERS MENU')
-    #         print ('0. Return to the MAIN MENU.')
-    #         print ('1. To PRINT all orders')
-    #         print ('2. To CREATE a new order') 
-    #         print ('3. To UPDATE an order')
-    #         print ('4. To DELETE an order')
-    #         print('') 
-
-    #         orders_menu_choice = int(input(''))
-
-    #         if orders_menu_choice == 0:
-    #             break
+    if choice == 2:
+        while True:
+            print ('')
```

Comment:
Could you use a multiline string here instead of using multiple `print` calls?

Reply:
Yes, this would be better but for the initial readability I wanted it to be as easy as possible. The idea was to then start refactoring after week 4.